hls-visualization.ipynb

```diff
@@ -0,0 +1,720 @@
+{
+ "cells": [
+  {
+   "cell_type": "markdown",
+   "metadata": {},
+   "source": [
+    "# Multi-Band Visualization Preview for Harmonized Landsat Sentinel-2 (HLS) with Common Query Language (CQL2-JSON)\n",
+    "This notebook demonstrates how to\n",
+    "1. Identify available dates within a bounding box defining an event area for a given collection\n",
+    "2. Register a dynamic tiler search for an event and specific date range for a given collection\n",
+    "3. Explore different options for displaying multi-band harmonized Landsat and Sentinel assets with the raster-api\n",
+    "\n",
+    "**About the Data**\n",
+    "A small subset of HLS data has been ingested to the VEDA datastore to explore visualizations using the raster API ([pgstac-titiler](https://github.com/stac-utils/titiler-pgstac)). This limited subset includes a single granule each for visualizing changes before and after Hurricane Maria in 2017 and Hurricane Ida in 2021. Note about HLS datasets: The Sentinel and Landsat assets have been harmonized and can be used interchangeably in algorithms but the individual band assets are specific to each provider. This notebook focuses on displaying HLS data with a dynamic tiler so separate examples are provided for rendering the unique band assets of each collection.\n",
+    "\n",
+    "**Additional Resources**\n",
+    "\n",
+    "[HLSL30 Dataset Landing Page](https://lpdaac.usgs.gov/products/hlsl30v002/)\n",
+    "\n",
+    "[Landsat 8 Bands and Combinations Blog](https://www.l3harrisgeospatial.com/Learn/Blogs/Blog-Details/ArtMID/10198/ArticleID/15691/The-Many-Band-Combinations-of-Landsat-8)\n",
+    "\n",
+    "[HLSS30 Dataset Landing Page](https://lpdaac.usgs.gov/products/hlss30v002/)\n",
+    "\n",
+    "[Sentinel 2 Bands and Combinations Blog](https://gisgeography.com/sentinel-2-bands-combinations/)\n",
+    "\n",
+    "[CQL2 STAC-API Examples](https://github.com/radiantearth/stac-api-spec/tree/master/fragments/filter#examples)\n",
+    "\n",
+    "- Author: Alexandra Kirk\n",
+    "- Last Updated Date: June 30, 2022"
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": 1,
+   "metadata": {},
+   "outputs": [],
+   "source": [
+    "import json\n",
+    "import requests\n",
+    "from folium import Map, TileLayer"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "metadata": {},
+   "source": [
+    "## Exploring the stac catalog - curated parameters for investigating specific hurricane events with the dynamic tiler and custom band combinations\n",
+    "Notebook examples focus on HLS S30 data for Hurricane Ida but Hurricane Maria and LS30 parameters are provided below for further exploration"
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": 2,
+   "metadata": {},
+   "outputs": [],
+   "source": [
+    "# Endpoints\n",
+    "stac_base_url = \"https://staging-stac.delta-backend.com\"\n",
```

Comment:
minor but I was trying to be consistent in using `STAC_API_URL = ...` across notebooks